react-intl/index.d.ts

```diff
@@ -227,7 +228,9 @@ declare namespace ReactIntl {
             defaultFormats?: Object;
         }
     }
-    class IntlProvider extends React.Component<IntlProvider.Props, any> { }
+    class IntlProvider extends React.Component<IntlProvider.Props, any> {
+        getChildContext(): InjectedIntlProps;
```

Comment:
Oh, not, you are correct, now its type for the whole context, not just `intl` part of it.

However I am thinking if it is safe to do it like this. Still `InjectedIntlProps` is separate thing and the fact that it seems to work here for now might be just a coincidence. So I suggest to do it like:

```ts
getChildContext() {
  return { intl: InjectedIntl };
}
```